### DISCUSSION TITLE
am i going nuts

### DISCUSSION THREAD

**Discussion by umop3plsdn** [manna-harbour/miryoku#281] [Q&A] [answered]
where is the tab key and ? on the corne

**Comment by manna-harbour** [comment, marked as answer]
They aren't obvious on the main diagram, as `Tab` is represented by `↹`, and `?` is not shown as it's just shifted `/`.

For details see: https://github.com/manna-harbour/miryoku/tree/master/docs/reference#base.

You might still be going nuts though.

**Comment by umop3plsdn** [comment]
wow i feel silly now but thanks
